app/views/events/show.html.erb

```diff
@@ -36,3 +36,36 @@
 
 <%= link_to 'Edit', edit_event_path(@event) %> |
 <%= link_to 'Back', events_path %>
+
+<%= form_with model: [@event, @comment] do |f| %>
+  <%= f.text_field :body %>
+  <%= f.submit "comment", disable_with: 'commenting...' %>
+<% end %>
```

Comment:
Please separate this logic as is not for "show events", a partial is ok

Reply:
I have finished making the change